### DISCUSSION TITLE
Win 11 mit USB Stick installieren

### DISCUSSION THREAD

**Discussion by alphabravocharly72** [R3NE07/Futro-S740#35] [General]
Hallo,
hoffe ihr könnt mir weiter helfen.
Habe im Netz dazu nichts gefunden.
Habe auf den S740 das letzte Bios Update installiert.
Nun möchte ich gern win 11 installieren. Eine 128GB M2 ist schon installiert.
Habe eine Win 11 iso datei auf einen USB Stick kopiert.
Leider findet der S740 diesen nicht um von dort aus zu booten. USB ist alles enabled. Secure boot off.
SSD ist  mit GPT formatiert.
Könnt ihr mir hier weiter helfen wie ich jetzt vorgehen muss.
Recht schönen Dank
alpha

**Comment by danzbox** [comment]
Die Iso zu kopieren reicht nicht. Der Stick muss schon bootfähig sein und das Windows Install menu laden.
Am einfachsten geht das für reine Windows Anwender mit dem Windows Media creation tool.
https://go.microsoft.com/fwlink/?linkid=2156295
